jme3-core/src/main/java/com/jme3/material/MatParam.java

```diff
@@ -301,20 +302,17 @@ public void write(JmeExporter ex) throws IOException {
         OutputCapsule oc = ex.getCapsule(this);
         oc.write(type, "varType", null);
         oc.write(name, "name", null);
-        if (value instanceof Savable) {
-            Savable s = (Savable) value;
-            oc.write(s, "value_savable", null);
+        if (value == null) {
+        } else if (value instanceof Savable) {
```

Comment:
Please don't use empty if statements.
I'd rather have an early return like 
```
if(value==null){
    return;
}
if(value instanceof Savable){
...
```
Or at least put the if/else chain in a if (value != null) statement.

Reply:
@Nehon it was a suggestion from IntelliJ IDEA. =/

Reply:
So? IDE's are dumb, you are in control of your code.